client/main.lua

```diff
@@ -256,96 +290,108 @@ end)
+
+                    currentGear.mask = CreateObject(maskModel, 1.0, 1.0, 1.0, 1, 1, 0)
+
+                    local bone2 = GetPedBoneIndex(cache.ped, 12844)
+
+                    AttachEntityToEntity(currentGear.mask, cache.ped, bone2, 0.0, 0.0, 0.0, 180.0, 90.0, 0.0, 1, 1, 0, 0, 2, 1)
+
+                    SetEnableScuba(cache.ped, true)
+                    SetPedMaxTimeUnderwater(cache.ped, 2000.00)
+
+                    currentGear.enabled = true
+
+                    ClearPedTasks(cache.ped)
+
+                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+
+                    oxgenlevell = oxgenlevell
+
+                    CreateThread(function()
+                        while currentGear.enabled do
+                            if IsPedSwimmingUnderWater(cache.ped) then
+                                oxgenlevell = oxgenlevell - 1
+
+                                if oxgenlevell == 90 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 80 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 70 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 60 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 50 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 40 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 30 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 20 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 10 then
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
+                                elseif oxgenlevell == 0 then
+                                    SetEnableScuba(cache.ped, false)
+                                    SetPedMaxTimeUnderwater(cache.ped, 1.00)
+
+                                    currentGear.enabled = false
+                                    iswearingsuit = false
+
+                                    TriggerServerEvent("InteractSound_SV:PlayOnSource", nil, 0.25)
                                 end
```

Comment:
```suggestion
if oxgenlevell % 10 == 0 and not oxgenlevell == 100 then
    TriggerServerEvent("InteractSound_SV:PlayOnSource", "breathdivingsuit", 0.25)
elseif oxgenlevell == 0 then
    SetEnableScuba(cache.ped, false)
    SetPedMaxTimeUnderwater(cache.ped, 1.00)

    currentGear.enabled = false
    iswearingsuit = false

    TriggerServerEvent("InteractSound_SV:PlayOnSource", nil, 0.25)
end
```

The modulo ```%``` operator exists and can prevent these huge ```elseif``` trees of repeating code.